tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -310,10 +317,18 @@ def fit(train_data,
             where `L` ranges from 0 to 50 (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels)
 
         Kwargs can include additional arguments for advanced users:
+            AG_args : dict, default={}
```

Comment:
naming: why `AG_` prefix? Should this be more descriptive: something like  `all_models_args`?

Reply:
to avoid namespace pollution. This shares namespace with arbitrary model hyperparameters, so we want to ensure it is uniquely identifiable to AutoGluon.